db/migrate/20161015161617_add_year_to_models.rb

```diff
@@ -0,0 +1,11 @@
+class AddYearToModels < ActiveRecord::Migration
+  def change
+  	add_column :hosts, :year, :string
+  	add_column :guests, :year, :string
+  	add_column :witnesses, :year, :string
+
+  	Host.update_all year: '2016'
+  	Guest.update_all year: '2016'
+  	Witness.update_all year: '2016'
```

Comment:
if run migration later on, can corrupt data with wrong year, since some would already be current year(2017)